### ISSUE TITLE
Beautify breaks when js starts with space followed by multi-line comment

### ISSUE THREAD

**Issue by varunkumar94** [beautify-web/js-beautify#789]
the following code beautify is looping itself

``` javascript
// original
    /*
    My Comment....
    */
    var p = 5+6;

//1st iteration
/*
        My Comment....
        */
    var p = 5 + 6;

//2nd iteration
/*
            My Comment....
            */
    var p = 5 + 6;

//nth iteration
    /*
                        My Comment....
                        */
    var p = 5 + 6;

```

**Comment by bitwiseman** [comment]
Input:
```js
 // my comment
 /*
 My Comment....
 */
 var p = 5 + 6;
 // my comment
 /*
 My Comment....
 */
 var p = 5 + 6;
```
Expected:
```js
 // my comment
 /*
 My Comment....
 */
 var p = 5 + 6;
 // my comment
 /*
 My Comment....
 */
 var p = 5 + 6;
```
Actual:
```js
 // my comment
 /*
  My Comment....
  */
 var p = 5 + 6;
 // my comment
 /*
 My Comment....
 */
 var p = 5 + 6;
```

**Comment by MacKLess** [comment]
Just a little more information. When I ran this in v.1.8.4 the actual output was the same as immediately above. When I removed the first comment, the second comment (block comment) indented correctly. When replacing the initial inline comment with another block comment, the same incorrect indentation behavior was seen:

Actual Output: 
```
       /*
     comment b
   */
   /*
        comment b
      */
   var p = 5 + 6;
   // comment c
   /*
     comment d
   */
   var p = 5 + 6;
```

We might need to add a special case dealing with block comments following a prior comment.
